.circleci/config.yml

```diff
@@ -234,6 +234,30 @@ jobs:
       - store_test_results:
           path: /go/out/tests
 
+  run-minimal-test:
+    <<: *integrationRemote
+
+    # Can run multiple images, first is primary, all use localhost
+    steps:
+      - checkout
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - run: make gitdep
+      - run:
+          # Create KIND cluster (separate so we get time info)
+          command: make info dep prepare
+      - run:
+          command: make sync
+      - run:
```

Comment:
Generally we split the install of a profile from running the test - mainly to make it easier to see the time and
where the failure happens.

Reply:
done: https://github.com/istio/installer/pull/143/commits/60a59f5563619c4f6d13457f452c7ca5ed42b9b3